segregation/singlegroup/absolute_clustering.py

```diff
@@ -58,18 +56,14 @@ def _absolute_clustering(data, group_pop_var, total_pop_var, alpha=0.6, beta=0.5
     t = data[total_pop_var].values
     n = len(data)
 
-    maxdist = np.max(
-        euclidean_distances(
-            pd.DataFrame({"x": data.centroid.x.values, "y": data.centroid.y.values})
-        )
+    w = euclidean_distances(
```

Comment:
There quite a few files that have the following:

```python
w = euclidean_distances(
    pd.DataFrame({"x": data.centroid.x.values, "y": data.centroid.y.values})
)
w = w / w.sum(axis=1)
dist = np.exp(-w)
```

Might this be better sourced from a single `util` function?

Reply:
yeah, probably should